microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -92,8 +92,33 @@ Previously, you could not query for the etcd version included with {microshift-s
 //==== Administrator Perspective
 //admin perspectives go here
 
-//[id="microshift-4-15-networking"]
-//=== Networking
+//[id="microshift-4-15-security"]
+//=== Security and compliance
+//
+// This content will be added post-GA, as it is asynchronous content.
+
+[id="microshift-4-15-networking"]
+=== Networking
+
+[id="microshift-4-15-networking-doc-enhancements"]
+==== Networking documentation enhancements
+
+Various networking documentation improvements are now available in the {microshift-short} release.
+
+* *Network features with their customization status.* A new detailed table of the networking features and their customizations that are accessible in a {microshift-short} instance are described in xref:../microshift_networking/microshift-cni.adoc#microshift-nw-customization-matrix_microshift-about-ovn-k-plugin[MicroShift networking customization matrix].
+
+* *Network topology updates*. Extensive examples of the Network topology available in your {microshift-short} instance is also updated, see xref:../microshift_networking/microshift-cni.html#microshift-network-topology_microshift-about-ovn-k-plugin[Network topology].
```

Comment:
Suggestion - something reads a tad off here. Is it the examples that have been updated? If so, I'd make this change:
```suggestion
* *Network topology updates*. Extensive examples of the Network topology available in your {microshift-short} instance are also updated, see xref:../microshift_networking/microshift-cni.html#microshift-network-topology_microshift-about-ovn-k-plugin[Network topology].
```

Reply:
Yeah the `are` is better, thank you!